### PR TITLE
#843 response requisitions queries 2

### PR DESCRIPTION
**Pull request by joshxg**
#843 

- This is getting the list view page compiling / working
- Took away the app bar button for creating a customer requisition as manual ones are out of scope

### CODE REVIEW

File: packages/requisitions/src/CustomerRequisition/ListView/AppBarButtons.tsx [hunk 1]
```diff
@@ -1,32 +1,21 @@
...
   const t = useTranslation('common');
 
   return (
     <AppBarButtonsPortal>
       <Grid container gap={1}>
-        <ButtonWithIcon
-          Icon={<PlusCircleIcon />}
-          label={t('label.new-requisition')}
-          onClick={() => onCreate(true)}
-        />
```
Comment (joshxg) [inline packages/requisitions/src/CustomerRequisition/ListView/AppBarButtons.tsx:29]: out of scope!

File: packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx [hunk 2]
```diff
@@ -1,113 +1,49 @@
...
-    getCustomerRequisitionListViewApi(api)
-  );
+  const { data, onChangeSortBy, sortBy, onChangePage, pagination, filter } =
+    useResponseRequisitions();
 
-  const columns = useColumns<RequisitionRow>(
+  const columns = useColumns<ResponseRequisitionRowFragment>(
     [
-      [getNameAndColorColumn(), { setter: onUpdate }],
+      [getNameAndColorColumn(), { setter: () => {} }],
```
Comment (joshxg) [inline packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx:24]: need to add an updater fn

File: packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx [hunk 3]
```diff
@@ -1,113 +1,49 @@
...
-            }
-          };
-
-          createRequisition();
-        }}
-      />
-
-      <Toolbar onDelete={onDelete} data={data} filter={filter} />
-      <AppBarButtons onCreate={setOpen} />
+      <Toolbar onDelete={() => {}} data={data?.nodes} filter={filter} />
```
Comment (joshxg) [inline packages/requisitions/src/CustomerRequisition/ListView/ListView.tsx:39]: need to add a delete fn also.. need a `batch` end point

**Review by mark-prins — APPROVED** [review]
👍  cool - that looks good to me!

**Pull request merged**